autode/solvent/solvents.py

```diff
@@ -16,45 +17,94 @@ def get_solvent(solvent_name: str,
         solvent_name (str): Name of the solvent e.g. DCM. Not case sensitive
 
     Keyword Arguments:
-        implicit (bool): Implicit solvent. Default = True
+        implicit (bool): Implicit solvent.
 
-        explicit (bool): Explicit solvent. Default = False
+        explicit (bool): Explicit solvent.
+
+        num (int): Number of explicit solvent molecules to include in the
+                   explicit solvent
 
     Returns:
         (autode.solvent.solvents.Solvent | None)
 
     Raises:
         (ValueError): If both explicit and implicit solvent are selected
     """
-    if solvent_name is None or not (implicit or explicit):
+    # Must be either implicit or explicit solvent
+    implicit = kwargs.get('implicit', False)
+    explicit = kwargs.get('explicit', False)
```

Comment:
The problem with this is that it is not clear from `get_solvent`'s definition what keywords are expected and what are their default. This also makes it "easier" for users to reach exceptions since they _could_ call `get_solvent_name("water", implicit=True, explicit=True)`. Also in the current state of things `get_solvent_name("water")` raises a `ValueError`.

What about `def get_solvent_name(solvent_name, explicit = False)` ? Since the solvation has been assumed to be implicit everywhere until now, let's have it as the default. I'm expected it to be pretty clear to anyone using this function directly that not specifying `explicit=True` will result in implicit solvation. But you could also make it clear in the docstring.

If you really want the method type to be specified explicitly, then maybe make it a positional argument 
```
def get_solvent_name(solvent_name: str, method: str)
    """
    # ...
    """
    if method not in ("implicit", "explicit"):
        raise ValueError("method must be either 'implicit' or 'explicit')
```

If think either of these would make the behavior clearer and reduce the number of possible exceptions (thus reducing the amount of safeguarding required)